### ISSUE TITLE
Very noticeable sluggishness when scrolling

### ISSUE THREAD

**Issue by richieb** [Polymer/polymer#1084]
My Polymer app has just text, HTML, and CSS (no images), similar to the official Polymer app at link below (which has hardly any images as well). And just as the scrolling on the official Polymer page blunderingly, lethargically struggles its way down the page, so too does my svelte app gracelessly reveals its contents as the user scrolls.
https://www.polymer-project.org/docs/polymer/layout-attrs.html

This sluggish scrolling, my only major concern with Polymer so far, has almost taken away all the fun I am having while developing my Polymer app.

In my former dev career as a  Flash/Flex developer, a time of yore when Steve Jobs was alive and healthy enough to single-handedly kill Flash with one blow to the head, buttery smooth, silky sexy animations and navigations were not a rarity but the norm. So I am hopeful we can replicate the same with JS **always**. Sure, Polymer's overall animations can impress and delight even the boring few who hate animations. But still, the sluggish scrolling has to be remedied somehow so Polymer can impress without this noticeable split in its pants crotch, as it performs on center stage in prime time.

**Comment by IntranetFactory** [comment]
For me the page scrolls smooth. IE11, iPhone, even on an old iPad2.

**Comment by richieb** [comment]
On my Galaxy Note 4, I just tested the scrolling issue on the Polymer site, and I noticed there is noticeable albeit palatable sluggishness there as well. The scrolling is about 50% smoother on my Note 4 than on my retina MacBook pro. Now I am wondering if this is an issue with retina MacBook pros, particularly since I just Googled and learned that some people have experienced similar performance issues on their retina machines.

If this is not a Polymer performance issue, I will be the 21,000,435 happiest person on earth today. I have worked too hard and invested too much in Polymer to have it poke me in the eye like this. So I am actually hoping the issue is not related to Polymer core.

Am I the only person experiencing this sluggish scrolling on the Polymer site? As I noted, my app has exactly the same problem.

**Comment by ebidel** [comment]
The site scrolls great for me on Chrome Android and FF mobile (N5).

We're not doing anything special with scrolling on the site. There's some
logic that sticks the top nav on mobile, but that's done once on page load.

If what you're seeing is slowness in native browser scrolling, there's not
much Polymer can do to improve that. What browser are you using on the Note
4?

**Comment by richieb** [comment]
Well, I just tested a few sites and I noticed that nearly every site I test stutters on my retina laptop. I originally noticed the stuttering on my Polymer app because I guess I was staring at it too long and too often :).

I would admit, with my background creating sexy Flash animations back in day (just a few years ago, actually), I do have a keen pair of eyes for jittery animations, so perhaps the stuttering looks much worse to me than it looks to most people. In any event, since the issue doesn't appear to be a Polymer issue, we can consider this issue closed. And I am a happy man again.
